src/sage/combinat/root_system/root_space.py

```diff
@@ -328,7 +328,7 @@ def associated_coroot(self):
 
     def quantum_root(self):
         r"""
-        Returns True if ``self`` is a quantum root and False otherwise.
+        Return ``True`` if ``self`` is a quantum root and ``False`` otherwise.
```

Comment:
Why not changing to a formulation like `Check whether ``self`` is a quantum root`